src/preact/base-element.js

```diff
@@ -477,3 +505,55 @@ function matchChild(element, defs) {
   }
   return null;
 }
+
+/**
+ * @param {!NodeList} nodeList
+ * @return {boolean}
+ */
+function shouldMutationForNodeListBeRerendered(nodeList) {
+  for (let i = 0; i < nodeList.length; i++) {
+    const node = nodeList[i];
+    if (node.nodeType == /* ELEMENT */ 1) {
+      // Ignore service elements, e.g. `<i-amphtml-svc>` or
+      // `<x slot="i-amphtml-svc">`.
+      if (
+        startsWith(node.tagName.toLowerCase(), IGNORE_NAME) ||
```

Comment:
Nit: Change `IGNORE_NAME`'s value to uppercase, and remove the repeated `toLowerCase()`
```suggestion
        startsWith(node.tagName, IGNORE_NAME) ||
```

Reply:
Done